slam3d/sensor/pcl/PointCloudSensor.hpp

```diff
@@ -138,6 +138,11 @@ namespace slam3d
 		                                         const Transform& odometry,
 		                                         bool loop);
 		
+		Transform align(PointCloudMeasurement::Ptr source,
```

Comment:
I think this method should not be part of the public interface. From outside, createConstraint() is the correct method to access this functionality.